stable/velero/templates/volumesnapshotlocation.yaml

```diff
@@ -11,14 +11,12 @@ spec:
 {{- with .Values.configuration }}
 {{- with .volumeSnapshotLocation }}
   provider: {{ .name  }}
-  objectStorage:
-    bucket: {{ .bucket  }}
 {{ with .config }}
   config:
   {{- with .region }}
     region: {{ . }}
   {{- end }}
-  {{- with .apitimeout }}
+  {{- with .apiTimeout }}
```

Comment:
This looks like a breaking change. Perhaps we could hold off on it until the next major release, or add some logic to the template to handle either value.

Also if this is only required for Azure, then it would be possible to make missing values a templating error rather than a potential runtime error (and give chart users better feedback) by doing something like:

```yaml
{{- if eq .name "azure" }}
  apiTimeout: {{ required "configuration.volumeSnapshotLocation.config.apiTimeout is required for azure volumeSnapshotLocation" .apiTimeout }}
{{- end }}
```

Reply:
fair point - the `values.yaml` and documentation use `apiTimeout`, so I was updating it to match, but I suppose it should go the other way around for this PR. 

I'm not familiar with the `required` syntax, but I can look into it.

Reply:
reverted breaking change in latest commit, and updated docs & values.yaml to reflect lowercased field name